web/src/app/pages/tasks/viewtasks.controller.js

```diff
@@ -4,7 +4,7 @@
     angular.module('BlurAdmin.pages.viewtasks').controller('ViewTasksPageCtrl', ViewTasksPageCtrl);
 
 
-    function ViewTasksPageCtrl($scope, $timeout, $http, AuthenticationService, toastr, toastrConfig) {
+    function ViewTasksPageCtrl($scope, $timeout, $http, AuthenticationService, toastr, toastrConfig, datas, data) {
```

Comment:
Use meaningful names when defining variables, input args, etc. instead of `datas, data`.